test/unit/node/routes/static.test.ts

```diff
@@ -7,7 +7,7 @@ import * as integration from "../../../utils/integration"
 
 const NOT_FOUND = {
   code: 404,
-  message: "not found",
+  message: "Not Found",
```

Comment:
🛠️ Change needed to make test pass (casing shouldn't matter). I was going to use `.toMatch(/string/)` but this was quicker